rotkehlchen/chain/evm/tokens.py

```diff
@@ -201,6 +208,37 @@ def _query_chunks(
             total_token_balances = combine_dicts(total_token_balances, new_token_balances)
         return total_token_balances
 
+    def _compute_detected_tokens_info(self, addresses: list[ChecksumEvmAddress]) -> DetectedTokensType:  # noqa: E501
+        addresses_info = {}
+        with self.db.conn.read_ctx() as cursor:
+            for address in addresses:
+                tokens, last_queried_timestamp = self.db.get_tokens_for_address(
```

Comment:
Using the proper name would make the code easier to read

```suggestion
                detected_tokens, last_queried_timestamp = self.db.get_tokens_for_address(
```